cmd/log-shuttle/main.go

```diff
@@ -244,8 +244,9 @@ func main() {
 
 	go LogFmtMetricsEmitter(s.MetricsRegistry, config.StatsSource, config.StatsInterval, s.Logger)
 
-	// Blocks until os.Stdin is closed
+	// Blocks until os.Stdin errors
 	s.ReadLogLines(os.Stdin)
```

Comment:
Because I want to know when it errors out.
I would also need to add another method on shuttle to block until all readers return.
Think I should do that?